src/Post.php

```diff
@@ -118,7 +118,7 @@ public function topic()
      */
     public function user()
     {
-        return $this->belongsTo(User::class);
+        return $this->belongsTo(config('canvas.model', User::class));
```

Comment:
I think with the above change, we shouldn't need to pass a default class here.

Reply:
We should always provide a default, in case a dev sets the config to null. I agree with providing `Illuminate\Foundation\Auth\User::class` as the default, but we need to guard against a `null` value here.